setup.cfg

```diff
@@ -71,7 +71,7 @@ install_requires =
     ansible-lint >= 4.0.2, < 5
 
     anyconfig == 0.9.7
-    cerberus >= 1.2, < 3
+    cerberus <= 1.3
```

Comment:
Maybe link to https://github.com/ansible/molecule/issues/2022#issuecomment-488682442 in a comment?